lens/core/view-set.scrbl

```diff
@@ -22,3 +22,23 @@
   @lenses-examples[
     (lens-set first-lens '(1 2 3) 'a)
 ]}
+
+@defproc[(lens-view* [target target/c] [lens lens?] ...) view/c]{
+  Like @racket[lens-view], except that it can take multiple lenses.
```

Comment:
I think appending "which are composed into a nested lens" to this sentence would be good explicitness.